recipes-core/glibc-locale-tests/glibc-locale-tests/test_locale_aliases.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# SPDX-License-Identifier: MIT
+set -euo pipefail
+
+. /usr/lib/ptest-utils/bash/ptest-format.sh
+
+# Expect these aliases on a Base System Image install.
+EXPECTED_ALIASES=("CP932" "CP936" "L1")
+
+expected_aliases_on_system() {
+    readarray -t locales_and_aliases < <(locale -a | tr "\n" " ")
+    readarray -t installed_locales < <(localedef --list-archive | tr "\n" " ")
+    readarray -t aliases < <(echo ${locales_and_aliases[@]} ${installed_locales[@]} | tr " " "\n" | sort | uniq -u | tr "\n" " ")
```

Comment:
Happy to add comments explaining this, but basically:
`locale -a` will list all valid locales including aliases.
`localedef --list-archive` will only list the actual installed locales and NOT the aliases.